src/drawing/backend_impl/bitmap.rs

```diff
@@ -165,6 +175,27 @@ impl<'a> BitMapBackend<'a> {
         }
     }
 
+    /// Helper function to convert with_buffer output
+    ///
+    /// This is a simple helper function which converts RGB triples
+    /// to RGB 32 bit values. Useful when interoprtating with crates
+    /// which expect Vec<32> such as minifb
+    /// Example: 
+    /// ```
+    /// let disp_buff: Vec<u32> = raw_bit_buffer.chunks_exact(3).map(|rgb| rgb_from_u8_to_u32(
```

Comment:
Also, this makes `cargo test` fails.

Reply:
Is it? I had tests fail on me locally, but they were
```
test drawing::area::drawing_area_tests::test_split_grid ... ok
test chart::context::test::test_chart_context ... FAILED
test style::font::ttf::test::test_font_cache ... FAILED
test chart::builder::test::test_caption ... ok
test drawing::backend_impl::bitmap::test_draw_line_out_of_range ... ok
thread panicked while panicking. aborting.
```

I've removed the rgb decode function and the comments and those tests still fail.

Reply:
This seems due to doc test failing. Another threading issue with the test case is fixed this morning. That's mine. 😂

Reply:
Gotcha. I looked at my test failure and it was trying to run the code in the comment. My face was like 🤔